packages/components/src/components/notification/_inline-notification.scss

```diff
@@ -136,6 +136,31 @@
     word-break: break-word;
   }
 
+  .#{$prefix}--inline-notification__action-button {
+    @if $carbon--theme == $carbon--theme--white {
```

Comment:
Since this component is "inverted" and there's not really a good inverted `$link-01` color, this was my approach for inverting theme token colors 🤔

Reply:
Hmmm I’m not sure this scales really well if we add a
new theme or if someone uses their own theme. I think either using carbon—blue-40 or making a new token would be appropriate.

Reply:
>  I think either making an inverted link token would be appropriate.

What was the "or" here? For "either... or ..." ? 

---

The token approach is fine -- perhaps a general token for inverted link colors like `$inverted-link`. But it's worth noting that there's also the problem of the ghost button hover color. And either way I need some design input here.

@shixiedesign what do you think about a new invert link color token here, so that links appearing on an inverted background ui color work correctly?

Reply:
I went ahead and made an issue (https://github.com/carbon-design-system/carbon/issues/3329) and PR (https://github.com/carbon-design-system/carbon/pull/3330) for this new color token 🤔